railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -14,6 +14,11 @@ Rails.application.configure do
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Aborts server start when missing the key to decrypt secrets in
+  # `config/secrets.yml.enc`. Assign the encryption key to
+  # `ENV["RAILS_MASTER_KEY"]`.
+  config.raise_on_missing_encryption_key = true
```

Comment:
Let's go with `ensure_presence_of_master_key_for_encrypted_secrets`. There are many types of encryption keys. Might as well be as literal as we can.